jetty-core/jetty-session/src/main/java/org/eclipse/jetty/session/SessionHandler.java

```diff
@@ -140,50 +139,4 @@ public boolean process(Handler handler, Response response, Callback callback) th
             return handler.handle(this, _response, callback);
         }
     }
-
```

Comment:
What was the reason for removing this? Are you sure it's ok to return null for the `newSessionAPIWrapper`?

Reply:
I moved this class from a test class to a normal class that can be used directly.  When using core sessions directly there is no API wrapper and if you ask for the API you get null back.